dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -195,3 +218,43 @@ properties:
         Selects the pulsed mode for data-ready interrupt when enabled,
         and the latched mode when disabled.
         Sets the corresponding DRDY_PULSED bit in the CTRL7 register.
+    
+    ff-duration:
+      type: int
+      default: 0x0
+      required: false
+      description: |
+        The freefall duration value represented in milliseconds.
+        If the accelerometer readings of the all axes are lower
+        than the freefall threshold value for the freefall duration long,
+        then a freefall trigger occurs. This value is 5 bits long in the
+        register and 1 LSB = 1 * 1/ODR.  
+      
+    ff-threshold:
+      type: int
+      default: 0x0
```

Comment:
Please document in the description why this particular default value was chosen

Reply:
comment written.